### ISSUE TITLE
Rework Game Activity text when an Option die attacks a Radioactive die

### ISSUE THREAD

**Issue by jimmosk** [buttonmen-dev/buttonmen#2206]
In http://www.buttonweavers.com/ui/game.html?game=23388 ifni, which has Ornery Mighty dice, attacked one of Calmon's Radioactive dice, and split. The game activity report read:
"Attacker oHz(20/30=20) showing 7 split into: oHz(20/30) which grew into oHz(20/30=12) showing 4, and oHz(20/30) which grew into oHz(20/30=12) showing 3"

I think it should instead read "Attacker oHz(20/30=20) showing 7 split into: oHz(10) which grew into oHz(12) showing 4, and oHz(10) which grew into oHz(12) showing 3" -- because after the split the new dice are no longer Option dice; keeping that (20/30) only confuses things.

**Comment by irilyth** [comment]
I think they are technically still Option dice, just like a Swing die that splits due to Radioactive (or shrinks due to Berserk, or changes size due to Weak or Mighty or anything else) is still a Swing die, I mean, technically the die is replaced by a new die of the new size, but we've historically considered the new die to be of the same "type" as the old one, and Option is a type of die (along with regular, Swing, and Twin, I believe).

**Comment by AdmiralJota** [comment]
If that's the case, shouldn't it say it "split into: oHz(20/30=10) which grew into oHz(20/30=12)", etc.?

**Comment by jimmosk** [comment]
I agree with AbmiralJota that at a minimum that ought to be spelled out.
But it just points up the contradiction of saying that it's still an Option(20/30) die. So I'd propose that when a die loses its Option, Swing, or Twin properties, it ceases to be an Option, Swing, or Twin die.

**Comment by irilyth** [comment]
Does an Option die ever have its Option property while it's in play?

**Comment by AdmiralJota** [comment]
Turbo (etc.) ones would, but those get lost with Radioactive.
